### PR TITLE
updated category for LA Agent Health workbook

### PR DESCRIPTION
**Pull request by MGoedtel**
## PR Checklist

* [ ] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.
* [ ] *if updating templates, it is helpful to post a screenshot of what the changes look like.*
* [ ] *if updating templates, ensure that your parameters and steps have meaningful names.*
* [ ] *if adding new templates, ensure that your parameters and columns have display names.*
* [ ] *if adding new templates, or changing items in a gallery, it's helpful to post a screenshot of what the gallery looks like now*

This PR is to stage "soft-delete" of the existing LA Agent Health workbook by removing the resource type (i.e. category) value in the settings.json file so that it no longer appears in the catalog.  The LA Agent Health workbook has been replaced by the new AzMon Agents Health workbook(s).

### CODE REVIEW

File: Workbooks/Azure Monitor - Workspaces/Agent Health/settings.json [hunk 1]
```diff
@@ -2,5 +2,5 @@
     "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
     "name":"Agent Health",
     "author": "Microsoft",
-    "galleries": [{ "type": "workbook", "resourceType": "Azure Monitor", "order": 200 }]
+    "galleries": [{ "type": "workbook", "resourceType": "", "order": 200 }]
```
Comment (yves-chan) [inline Workbooks/Azure Monitor - Workspaces/Agent Health/settings.json:5]: for this part we can just have:

`"galleries": []`

Also let's add a description here saying something like:

"description": "not shown in any gallery, this workbook is deprecated, please use Workbooks/Azure Monitor - Agents/Overview"

**Comment by MGoedtel** [comment]
#sign-off

File: Workbooks/Azure Monitor - Workspaces/Agent Health/settings.json [hunk 2]
```diff
@@ -2,5 +2,6 @@
     "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
     "name":"Agent Health",
     "author": "Microsoft",
-    "galleries": [{ "type": "workbook", "resourceType": "Azure Monitor", "order": 200 }]
+    "galleries": [],
+	"description": "not shown in any gallery, this workbook is deprecated, please use Workbooks/Azure Monitor - Agents/Overview"
```
Comment (yves-chan) [inline Workbooks/Azure Monitor - Workspaces/Agent Health/settings.json:6]: nit: this looks like it's slightly indented

**Review by yves-chan — APPROVED** [review]
Changes for category name will be in a different PR correct?

**Comment by MGoedtel** [comment]
@yves-chan - correct, I'm handling that in a separate PR.  Thanks.

File: Workbooks/Azure Monitor - Workspaces/Agent Health/settings.json (hunk 2 above)
Comment (MGoedtel) [reply Workbooks/Azure Monitor - Workspaces/Agent Health/settings.json]: Fixed in latest commit.

**Comment by MGoedtel** [comment]
#sign-off

**Pull request merged**